stubs/3.2/re.py

```diff
@@ -46,20 +46,12 @@ def findall(pattern: AnyStr, string: AnyStr,
 def finditer(pattern: AnyStr, string: AnyStr,
              flags: int = 0) -> Iterator[Match[AnyStr]]: pass
 
-@overload
-def sub(pattern: AnyStr, repl: AnyStr, string: AnyStr, count: int = 0,
-        flags: int = 0) -> AnyStr: pass
-@overload
-def sub(pattern: AnyStr, repl: Callable[[Match[AnyStr]], AnyStr],
+def sub(pattern: AnyStr, repl: Union[AnyStr, Callable[[Match[AnyStr]]]],
```

Comment:
I think you're missing the return type here for `Callable`. I think this is causing the failing tests.